lib/draggable_scrollbar.dart

```diff
@@ -602,27 +602,27 @@ class ArrowClipper extends CustomClipper<Path> {
 }
 
 class SlideFadeTransition extends StatelessWidget {
-  final Animation<double> animation;
+  final Animation<double>? animation;
```

Comment:
This can be non-nullable since it is reqired and we should prevent using ! in build function.